tests/proofs/dexter/dexter-spec.md

```diff
@@ -15,6 +15,89 @@ module DEXTER-SPEC
 
   claim <k> (now 0 => .) ~> #dexterCode(_) ... </k>
         <mynow> #Timestamp(0) </mynow>
+
+  claim <k> PUSH nat X:Int ; PUSH nat Y:Int ; PUSH nat Z:Int ; MUL ; EDIV ; IF_NONE {} { DUP ; CDR ; SWAP ; CAR ; PUSH nat 0 ; DIG 2 ; COMPARE ; EQ } => . ... </k>
+        <stack> .Stack => [ bool ?_ ] ; [ nat (Z *Int Y) /Int X ] ; .Stack </stack>
+    requires X >Int 0
+     andBool Y >=Int 0
+     andBool Z >=Int 0
```

Comment:
Could you please explain what it is about?

Reply:
This is a test case for the failure of Z3 on non-linear reasoning. This failed without the rewrites to `#mulMod` and `#mulDiv`.